android/src/main/java/io/invertase/googleads/ReactNativeGoogleAdsBannerAdViewManager.java

```diff
@@ -31,16 +32,17 @@
 import com.google.android.gms.ads.AdRequest;
 import com.google.android.gms.ads.AdSize;
 import com.google.android.gms.ads.AdView;
+import com.google.android.gms.ads.LoadAdError;
 import java.util.Map;
 import javax.annotation.Nonnull;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Please use the androidx annotations, you already have `androidx-annotation.NonNull` above, no need for jetbrains NotNull as well 
```suggestion
```